champions/src/main/java/me/mykindos/betterpvp/champions/champions/skills/skills/brute/sword/BlockToss.java

```diff
@@ -112,7 +112,11 @@ private double getChargePerSecond(int level) {
 
     @Override
     public double getCooldown(int level) {
-        return cooldown - (level - 1d) * cooldownDecreasePerLevel;
+        if (cooldown - (level - 1d) * cooldownDecreasePerLevel < 1) {
+            return 0;
+        } else {
+            return cooldown - (level - 1d) * cooldownDecreasePerLevel;
+        }
```

Comment:
This is likely caused by an issue with your skills.yml in plugins/Champions/skills/skills.yml you can delete this file to regenerate it

![image](https://github.com/Mykindos/BetterPvP/assets/5389944/247b2442-fb66-454f-b9fd-e36b67d17a74)